hack/validate/changelog-well-formed

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+changelogFile=${1:-CHANGELOG.md}
+
+if [ ! -r "$changelogFile" ]; then
+  echo "Unable to read file $changelogFile" >&2
+  false
```

Comment:
`false` is a bash builtin command that will exit with value 1: http://www.tldp.org/LDP/abs/html/internal.html#FALSEREF